### PR TITLE
About view

### PR DESCRIPTION
**Pull request by billyfung**
basic stuff for #4

### CODE REVIEW

File: episode/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{% load static %}
+<!doctype html>
+
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+
+    <title></title>
+    <meta name="description" content="">
+    <meta name="author" content="Jin Park">
```
Comment (jinpark) [inline episode/templates/base.html:10]: put yo name in here

File: episode/tests.py [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+        """
 
-# Create your tests here.
+        episode = Episode(name="""
+                extra long name that is past the max number of characters
+                extra long name that is past the max number of characters
+                extra long name that is past the max number of characters
+                """)
+        response = self.client.get(Episode.objects.all())
+        self.assertIs(response.status_code, 200)
```
Comment (jinpark) [inline episode/tests.py]: plz add new line after
https://robots.thoughtbot.com/no-newline-at-end-of-file

File: episode/urls.py [hunk 3]
```diff
@@ -2,5 +2,6 @@
 from episode import views
 
 urlpatterns = [
-    url(r'^$', views.home)
+    url(r'^$', views.home),
+    url(r'^about', views.about)
```
Comment (jinpark) [inline episode/urls.py]: I personally like to have the more specific things up top and more general things on the bottom

**Review by jinpark — COMMENTED** [review]
fix the new lines stuff

File: episode/templates/base.html (hunk 1 above)
Comment (billyfung) [reply episode/templates/base.html:10]: maybe later

**Comment by jinpark** [comment]
lgtm

**Pull request merged**